Xamarin.Forms.Platform.iOS/Forms.cs

```diff
@@ -411,6 +412,95 @@ public SizeRequest GetNativeSize(VisualElement view, double widthConstraint, dou
 				return Platform.MacOS.Platform.GetNativeSize(view, widthConstraint, heightConstraint);
 #endif
 			}
+
+			#region Remove with Essentials API
+			public AppTheme RequestedTheme
+			{
+				get
+				{
+#if __IOS__ || __TVOS__
+					if (!IsiOS13OrNewer)
+						return AppTheme.Unspecified;
+#if __XCODE11__
+					var uiStyle = GetCurrentUIViewController()?.TraitCollection?.UserInterfaceStyle ??
+						UITraitCollection.CurrentTraitCollection.UserInterfaceStyle;
+
+					switch (uiStyle)
+					{
+						case UIUserInterfaceStyle.Light:
+							return AppTheme.Light;
+						case UIUserInterfaceStyle.Dark:
+							return AppTheme.Dark;
+						default:
+							return AppTheme.Unspecified;
+					};
+#else
+					return AppTheme.Unspecified;
+#endif
+#else
+					return AppTheme.Unspecified;
+#endif
+				}
+			}
+
+#if __IOS__ || __TVOS__
+
+			static UIViewController GetCurrentUIViewController() =>
+				GetCurrentViewController(false);
+
+			static UIViewController GetCurrentViewController(bool throwIfNull = true)
+			{
+				UIViewController viewController = null;
+
+				var window = UIApplication.SharedApplication.KeyWindow;
```

Comment:
should this code use
https://github.com/xamarin/Xamarin.Forms/blob/30228a2f3587e1b8c10674076a4a6a64908a1c19/Xamarin.Forms.Platform.iOS/Extensions/UIApplicationExtensions.cs#L8

Reply:
Same as above; Essentials, but changed it for now to be sure.